charts/platform-service/values.yaml

```diff
@@ -1,14 +1,17 @@
 # Default values for platform-service
-
-# application name - required
-app: appname
+---
+# Application name
+app: app        # Required
 
 version: v1
 replicaCount: 1
 
 # sample extra label, you can add as many as you want
-#additionalPodLabels: |
-#  saml-idp-forward-auth-filter: enabled
+additionalPodLabels: {}

```

Comment:
I suggest we add additional field "globalLabels that can be used to label all resources generated by this chart (in the fashion that kubeIt.tags is used now but in a more transparent and generalized way). We can then drop kubeit.tags completely and just kubeit labels via this globalLabels field.

Reply:
Added a function in the helpers to handle the labels.